### DISCUSSION TITLE
[RELEASE] Teddy Survival

### DISCUSSION THREAD

**Discussion by MotoLegacy** [nzp-team/nzportable#321] [Map Releases]
## This is a manual migration of forum user oldschool125's map release `Teddy_Survival`. Below contains the original post content. You can view the original post [here](https://nzportable.forumotion.com/t2140-release-teddy-survival).

So lately I started working on this map on my free time .Here is some pictures : 
![0](https://github.com/nzp-team/nzportable/assets/26030250/212c4b8f-745a-4e8a-8f00-5764ef8ef14c)
![1](https://github.com/nzp-team/nzportable/assets/26030250/92d07ae7-4f2d-461b-9931-aaf286903207)
![2](https://github.com/nzp-team/nzportable/assets/26030250/c9a4fb47-f919-4cdb-b144-074fa1680348)
![3](https://github.com/nzp-team/nzportable/assets/26030250/4e38cc20-3882-4dcc-b70d-33102a98bc69)
![4](https://github.com/nzp-team/nzportable/assets/26030250/0b124094-aab9-4600-925e-f2bc20662b96)

Features:
All perks
Easter egg,pap....

Credits :

TrueNightmare - For letting me use his question marks wall buy

# [DOWNLOAD](https://github.com/nzp-team/nzportable/files/12147743/Teddy_Survival.zip)

**Comment by oshogaps-glitch** [comment]
this map is also bugged to most of the zombies are stuck in barricades like 90% only 1 or 2 percent actually come out most the time you will have to wait until they die and spawn in....

in the spawn room the litle dirt when zombies come up get stuck and start floating


also in the second rooom no zombies come out at all they just stuck in barricades
